src/views/Main/Catalog/Components/SchemaList.js

```diff
@@ -61,7 +65,54 @@ export class SchemaList extends React.Component {
   updateFn = () => {
     EntityDataModelApi.getAllSchemas()
       .then((schemas) => {
-        this.setState({ schemas: Utils.addKeysToArray(schemas) });
+        EntityDataModelApi.getAllPropertyTypes()
```

Comment:
same comment about `bluebird` and `Promise.join` to make these requests in parallel